src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -29,7 +29,7 @@
 public abstract class BaseUiTestCase extends BaseTestCaseWithBackDoorApiAccess {
 
     /** Indicates if the test-run is to use GodMode. */
-    private static boolean isGodModeEnabled;
+    private static boolean isGodModeEnabled = true;
```

Comment:
God mode is meant to be run only on your local PC. So going by [godmode.md](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md), what you have to do is -

- set `isGodModeEnabled = true;`
- run all CI tests on your PC.
- change `isGodModeEnabled = true;` to `isGodModeEnabled;`
- (Recommended step) Run CI tests again and ensure all tests pass.
- Now add, commit and push your files.